patches/server/0210-PlayerReadyArrowEvent.patch

```diff
@@ -28,12 +28,23 @@ index e088e5174a4666130cd1f66c751e7ab83d9f7a65..8aa5f8bacb868a3b04af6e696f9eed89
      @Override
      public ItemStack getProjectile(ItemStack stack) {
          if (!(stack.getItem() instanceof ProjectileWeaponItem)) {
-@@ -2242,7 +2253,7 @@ public abstract class Player extends LivingEntity {
+             return ItemStack.EMPTY;
+         } else {
+-            Predicate<ItemStack> predicate = ((ProjectileWeaponItem) stack.getItem()).getSupportedHeldProjectiles();
++            Predicate<ItemStack> predicate = ((ProjectileWeaponItem) stack.getItem()).getSupportedHeldProjectiles().and(item -> tryReadyArrow(stack, item)); // Paper
+             ItemStack itemstack1 = ProjectileWeaponItem.getHeldProjectile(this, predicate);
+ 
+             if (!itemstack1.isEmpty()) {
+                 return itemstack1;
+             } else {
+-                predicate = ((ProjectileWeaponItem) stack.getItem()).getAllSupportedProjectiles();
++                predicate = ((ProjectileWeaponItem) stack.getItem()).getAllSupportedProjectiles().and(item -> tryReadyArrow(stack, item)); // Paper
+ 
                  for (int i = 0; i < this.inventory.getContainerSize(); ++i) {
                      ItemStack itemstack2 = this.inventory.getItem(i);
  
 -                    if (predicate.test(itemstack2)) {
-+                    if (predicate.test(itemstack2) && tryReadyArrow(stack, itemstack2)) { // Paper
++                    if (predicate.test(itemstack2)) { 
```

Comment:
Looks like you added a space at the end of this line causing it to show up in the diff when it should be an "unchanged" line as you removed the diff that was originally added.

Reply:
Removed now, came from the paper comment that was originally there.